### ISSUE TITLE
Add in heat by my dad

### ISSUE THREAD

**Issue by ImNotAGdPlayer** [FlafyDev/auto-nong-indexes#305]
### Song Name

in heat

### Artist Name

my dad

### Source

Youtube

### Youtube Link

EBT9huP3ZsI

### Song ID

0

### Extra

Created from Auto Nong.
Created for level: 75322252

**Comment by FlafyDev** [comment]
I don't think we should replace stereo madness with it... the level should use a unused song to replace

**Comment by ImNotAGdPlayer** [comment]
hmm... ok try 1335298
